packages/manager/apps/telecom/package.json

```
@@ -23,6 +23,7 @@
     "@ovh-ux/manager-navbar": "^2.1.2",
     "@ovh-ux/manager-overthebox": "^5.0.0",
     "@ovh-ux/manager-sms": "^7.2.0",
+    "@ovh-ux/manager-telecom-spare": "^0.0.0",
```

```suggestion
    "@ovh-ux/manager-telecom-spare": "^0.0.0 || ^0.1.0",
```
or 
```suggestion
    "@ovh-ux/manager-telecom-spare": "^0.0.0 || ^1.0.0",
```
depending on whether you have a breaking change or not 
(if you have a 0.x.x a minor is considered as breaking)